src/adapter/aws-lambda/handler.ts

```diff
@@ -73,7 +73,12 @@ export const handle = <E extends Env = Env, S = {}, BasePath extends string = '/
 
 const createResult = async (res: Response): Promise<APIGatewayProxyResult> => {
   const contentType = res.headers.get('content-type')
-  const isBase64Encoded = contentType && isContentTypeBinary(contentType) ? true : false
+  const isBase64Encoded_contentType = contentType && isContentTypeBinary(contentType) ? true : false
```

Comment:
How about writing it like the following? This would reduce procedures:

```ts
  let isBase64Encoded = contentType && isContentTypeBinary(contentType) ? true : false

  if (!isBase64Encoded) {
    const contentEncoding = res.headers.get('content-encoding')
    isBase64Encoded = isContentEncodingBinary(contentEncoding)
  }
```

Reply:
this is more fast than my code.